l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -67,21 +76,25 @@ def __init__(self, path: str, key: str = ""):
         self.frames = np.empty(0, dtype=FRAME_DTYPE)
         self.scenes = np.empty(0, dtype=SCENE_DTYPE)
         self.agents = np.empty(0, dtype=AGENT_DTYPE)
+        self.tl_faces = np.empty(0, dtype=TL_FACE_DTYPE)
 
         # Note: we still support only zarr. However, some functions build a new dataset so we cannot raise error.
         if ".zarr" not in self.path:
             print("zarr dataset path should end with .zarr (for now). Open will fail for this dataset!")
         if not Path(self.path).exists():
             print("zarr dataset path doesn't exist. Open will fail for this dataset!")
 
-    def initialize(self, mode: str = "w", scenes_num: int = 0, frames_num: int = 0, agents_num: int = 0) -> None:
+    def initialize(
+        self, mode: str = "w", scenes_num: int = 0, frames_num: int = 0, agents_num: int = 0, tl_faces_num: int = 0
```

Comment:
`scenes_num`->`num_scenes`, likewise for the rest.
`num_scenes` is more usual, like `num_workers` in pytorch